### PR TITLE
Fix Plex config

### PR DESCRIPTION
**Pull request by Elod-T**
Changed the plex config, because you couldn't add any local libraries.
It is some bug by plex. But this workaround solves it.

### CODE REVIEW

**Comment by ivan-pinatti** [comment]
Hi @Elod-T,

The change you are doing it will actually route the network traffic differently because you are sharing the **host** network, and not going through the VPN.

Could you please elaborate a little bit more of the problem you are having and why this would solve it?

**Comment by Elod-T** [comment]
Hey!  My problem was that I couldn't add any media folders to the Plex server itself. It just showed me to download a plex media server. And it doesn't need to be routed through a VPN so I changed it to host and it solved the problem.

**Comment by ivan-pinatti** [comment]
@Elod-T 

That's odd. Shouldn't this be related to the Docker volume mount?
It looks to me more a folder mapping issue than a networking problem.

Are you mounting the right folders? Are the permissions correct?

Docker Documentation on volumes; https://docs.docker.com/storage/volumes/

**Comment by Elod-T** [comment]
@ivan-pinatti

Hello. No I can confirm it's not a mount issue, I didn't change any settings other than the network. The folders are working because the other services can work with them.

**Comment by ivan-pinatti** [comment]
@Elod-T, this is weird.
Why the network config is affecting the folders mounting? Could you please explain?

Also, you mentioned a bug, do you have the bug number or information? I would like to review it.

**Comment by clemone210** [comment]
I could not add libraries as well.
I needed to add the PLEX_CLAIM parameter to the docker-compose.yaml file.
It can be requested here: https://www.plex.tv/claim/

**Comment by ivan-pinatti** [comment]
Hi @Elod-T and @clemone210,

Are you still having this problem? I am working on a new release and everything seems to be working fine on my end.

**Comment by clemone210** [comment]
It only appears on a whole new setup.
If you do not have a server already setup, u first need to pass the PLEX_CLAIM env to the container in order to claim the server. Then you can access http://localhost:32400/ to run the "setup".
Once its claimed you can remove the env variable.

**Comment by ivan-pinatti** [comment]
@clemone210 I've did some updates to the code today. I ran a completely empty stack and it was 100% fine.

Can you try to reproduce the problem and check if that still happens? If it does, please let me know the steps so I can try on my machine.

**Comment by Elod-T** [comment]
Hey! I didn't create a new instance since, so I can't confirm it, sorry

**Comment by ivan-pinatti** [comment]
Thanks for the reply @Elod-T .

I will wait on @clemone210 to validate, otherwise I will close the PR.

**Comment by clemone210** [comment]
Thanks for the code update.
VPN works again.
Strange that I always need the Plex Claim token, otherwise it wont run for me.
Maybe its also an issue due to I always run on a VPS not at home and do not have browser access.

**Review by clemone210 — APPROVED** [review]
approved - running without error.
Thanks for the code update.
VPN works again.
Strange that I always need the Plex Claim token, otherwise it wont run for me.
Maybe its also an issue due to I always run on a VPS not at home and do not have browser access.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
+version: "3.8"
 services:
-
   plex:
     image: linuxserver/plex:${PLEX_VERSION}
     container_name: plex
-    depends_on:
-      - protonvpn
-    network_mode: service:protonvpn
+    network_mode: host
```
Comment (clemone210) [inline docker-compose.yml:6]: thats what I changed as well. Makes sense to have this on the host network

Comment (ivan-pinatti) [reply docker-compose.yml:6]: @clemone210 

In the new release (just published it today) I put it together with all the other containers as they can communicate in their "internal network" as localhost/127.0.0.1 .

Please let me know if the new version works and solves all the problems, lots of changes there.

Comment (ivan-pinatti) [reply docker-compose.yml:6]: Reverted to host in PR #16

**Comment by ivan-pinatti** [comment]
Problem solved in PR #16

**Pull request closed without merging**